### ISSUE TITLE
Power off menu.

### ISSUE THREAD

**Issue by ghostslayer989** [ShizukuIchi/winXP#25]
When someone clicks (Turn Off) on the "Turn off computer" menu it should close the tab, and when someone clicks (Restart) it should refresh the tab.

**Comment by maracuja-juice** [comment]
Some kind of power off seems to be already implemented:

```
const powerOffAnimation = keyframes`
  0% {
    filter: brightness(1) grayscale(0);
  }
  30% {
    filter: brightness(1) grayscale(0);
  }
  100% {
    filter: brightness(0.6) grayscale(1);
  }
`;
const animation = {
  [POWER_STATE.START]: '',
  [POWER_STATE.TURN_OFF]: powerOffAnimation,
  [POWER_STATE.LOG_OFF]: powerOffAnimation,
};
```

When is this code triggered?

**Comment by ghostslayer989** [comment]
Never right now. When I click "Turn Off" It shows the fake "C:\ Application not found"

**Comment by maracuja-juice** [comment]
Yes I know. I haven't found out yet by looking through the code where it would need to be added.

**Comment by ghostslayer989** [comment]
Me neither

**Comment by ShizukuIchi** [comment]
Sounds interesting! But I prefer showing blue screen or shut down... screen.
btw the code @maracuja-juice mentioned is for grayscaling whole app when power off menu is visible.

**Comment by ghostslayer989** [comment]
Hmm which bsod error code?

**Comment by ghostslayer989** [comment]
How about this one? Seeing as we don't really have "Acpi Drivers" due to it being a website.

> This BSOD means that the ACPI driver detected an internal inconsistency. BSOD error code 0x000000A3 may also show "ACPI_DRIVER_INTERNAL" on the same blue screen.

**Comment by maracuja-juice** [comment]
Something like this but for XP: https://screen.now.sh/win10-crash
